README.md

```diff
@@ -138,40 +139,46 @@ To run all the test cases with race detection:
     $ ginkgo -r -race
 
 
-<a name="Examples"></a>
+<a name="examples"></a>
 ## Examples
 
 A variety of example applications are provided in the [`examples`](examples) directory.
 
-<a name="Tools"></a>
+<a name="tools"></a>
 ### Tools
 
 A variety of clones of original tools are provided in the [`tools`](tools) directory.
 They show how to use more advanced features of the library to re-implement the same functionality in a more concise way.
 
-<a name="Benchmarks"></a>
+<a name="benchmarks"></a>
 ## Benchmarks
 
 Benchmark utility is provided in the [`tools/benchmark`](tools/benchmark) directory.
 See the [`tools/benchmark/README.md`](tools/benchmark/README.md) for details.
 
-<a name="API-Documentation"></a>
+<a name="api-documentation"></a>
 ## API Documentation
 
 A simple API documentation is available in the [`docs`](docs/README.md) directory. The latest up-to-date docs can be found in [![Godoc](https://godoc.org/github.com/aerospike/aerospike-client-go?status.svg)](https://pkg.go.dev/github.com/aerospike/aerospike-client-go/v7).
 
-<a name="App-Engine"></a>
+<a name="google-app-engine"></a>
 ## Google App Engine
 
 To build the library for App Engine, build it with the build tag `app_engine`. Aggregation functionality is not available in this build.
 
 
-<a name="Reflection"></a>
+<a name="reflection-and-object-api"></a>
 ## Reflection, and Object API
 
 To make the library both flexible and fast, we had to integrate the reflection API (methods with `[Get/Put/...]Object` names) tightly in the library. In case you wanted to avoid mixing those API in your app inadvertently, you can use the build tag `as_performance` to remove those APIs from the build.
 
 
+<a name="proxy-client--dbaas"></a>
+## Proxy Client / DBAAS
+
+To compile the client for the proxy server in out database as a service environment, pass `-tags as_proxy` to the compiler on build.
```

Comment:
Is this a typo?  "To compile the client for the proxy server in _our_ database as a service environment"